jobs/JOBSPROC_RAP_PREP_POST

```diff
@@ -115,7 +120,7 @@ export SENDDBN=${SENDDBN:-YES}
 # Paths for obsproc items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc_rap.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc_rap.${obsproc_ver:?\
```

Comment:
See same as JOBSPROC_RAP_*; also note the missing close "}". change to:
-${PACKAGEROOT:?}/${obsNET}/${obsproc_ver}:?\